odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewTags/OverviewTags.tsx

```diff
@@ -49,9 +45,9 @@ const OverviewTags: React.FC<OverviewTagsProps> = ({
             </AppButton>
           }
         />
-      </div>
+      </CaptionContainer>
       {tags?.length ? (
-        <div className={classes.tagsContainer}>
+        <TagsContainer>
```

Comment:
Here, instead of TagsContainer inherited from div you can use mui Box component and pass to him sx prop with margin styles